tests/software/cooler/cload/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { COOLER_CLOAD } from '../../../../software/cooler/cload/main.nf' addParams( options: [:] )
```

Comment:
We probably should add several more tests for each of the `tools` (`hiclib`, `tabix`, `pairs` and `pairix`), as an example I modified the current one for `pairix`. Note, that since you will have to include the module several time you will need to renamed it with `as`.

```suggestion
include { COOLER_CLOAD as COOLER_CLOAD_PAIRIX } from '../../../../software/cooler/cload/main.nf' addParams( options: ['args' : 'pairix'] )
```